runatlantis.io/docs/server-configuration.md

```diff
@@ -1266,6 +1274,8 @@ Setting this to `false` can be useful in an air-gapped environment where a downl
 
   This has no impact if `--tf-download` is set to `false`.
 
+  This setting is currently ignored if `--tf-distribution` is set to `opentofu`
```

Comment:
What is the rationale behind ignoring this for opentofu? Does this mean we always download the binary?

Reply:
I'm not sure exaclty how to support this flag with opentofu. I'm open to ideas.

Tofudl supports two fields to download from a mirror: https://github.com/opentofu/tofudl/blob/76c6857d6eac665824b29da4e893e06e0b1896b2/branding/branding.go#L10-L14. I think we'd need to add an additional flag at the very least? I'm also not sure how specific this setting is to tofudl, that might be an issue if we change download method.

Reply:
@meringu we can maybe ask on the opentofu slack or an issue to tofudl if neccesary

Reply:
Great idea, I have asked over here: https://opentofucommunity.slack.com/archives/C05RKE7PS72/p1726523077638359

Reply:
The thinking is that we could look at supporting this later once someone needs it and has defined use-case. For now, I've updated the wording to indicate that it is not yet supported, and can unblock us getting this in for others to start to try out OpenTofu without a mirror.